libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Expressions/Converters/DialogExpressionConverter.cs

```diff
@@ -32,8 +32,23 @@ public DialogExpressionConverter(ResourceExplorer resourceExplorer, SourceContex
             this.resourceExplorer = resourceExplorer;
         }
 
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="Newtonsoft.Json.JsonConverter"/> can read JSON.
+        /// </summary>
+        /// <value>
+        /// true.
```

Comment:
maybe add <c> tag to the true value, to represent its a value itself and that's why its not capitalized